Celestial/FigCaptureStillImageSettings.h

```diff
@@ -0,0 +1,22 @@
+#import <Foundation/Foundation.h>
+
+API_AVAILABLE(ios(8.0))
+@interface FigCaptureStillImageSettings : NSObject
+
+@property (assign, nonatomic) bool outputMirroring;
```

Comment:
Are the types in this file not BOOL, CGFloat and NS(U)Integer?

Reply:
These types appear to match [nst/iOS-Runtime-Headers@fbb634](https://github.com/nst/iOS-Runtime-Headers/blob/fbb634c78269b0169efdead80955ba64eaaa2f21/PrivateFrameworks/Celestial.framework/FigCaptureStillImageSettings.h) which are from a 64-bit device

Reply:
@uroboro Yes I'm certain about this.